components/toolbar-menu/index.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
I have an implementation that allows cycling through the menu options with the keyboard. I am mostly happy with it except that hitting escape deselects the editor as well as closing the menu.